src/deluge/model/song/song.cpp

```diff
@@ -3401,7 +3401,7 @@ void Song::addOutput(Output* output, bool atStart) {
 	}
 
 	// Must resync LFOs - these (if synced) will roll even when no activeClip
-	if (playbackHandler.isEitherClockActive() && this == currentSong) {
+	if (this == currentSong) {
```

Comment:
is this ok? I thought they need a clock to know the tempo to sync to

Reply:
The guard is now in resyncGlobalLFO(), so no need to check here.